src/webview/git-import/gitImportLoader.ts

```diff
@@ -276,18 +292,65 @@ function validateDevFilePath(event: any) {
 }
 
 function showError(event: any, location: string, message: string): void {
-    panel.webview.postMessage({
+    panel?.webview.postMessage({
         action: event.action,
         status: false
     });
-    if (message.indexOf('already exists') !== -1) {
-        vscode.window.showErrorMessage(`Folder already exists on the selected ${location.substring(0, location.lastIndexOf('\\'))}`);
-    } else {
-        vscode.window.showErrorMessage('Error occurred while cloning the repository. Please try again.');
+    if (!forceCancel) {
+        if (message.indexOf('already exists') !== -1) {
+            vscode.window.showErrorMessage(`Folder already exists on the selected ${location.substring(0, location.lastIndexOf('\\'))}`);
+        } else {
+            vscode.window.showErrorMessage('Error occurred while cloning the repository. Please try again.');
+        }
     }
 }
 
 function isGitURL(host: string): boolean {
-    return ['github.com','bitbucket.org','gitlab.com'].includes(host);
+    return ['github.com', 'bitbucket.org', 'gitlab.com'].includes(host);
 }
 
+function deleteFile(dir, file) {
+    return new Promise<void>(function (resolve, reject) {
+        var filePath = path.join(dir, file);
+        fs.lstat(filePath, function (err, stats) {
+            if (err) {
+                return reject(err);
+            }
+            if (stats.isDirectory()) {
+                resolve(deleteDirectory(filePath));
+            } else {
+                fs.unlink(filePath, function (err) {
+                    if (err) {
+                        return reject(err);
+                    }
+                    resolve();
+                });
+            }
+        });
+    });
+};
+
+function deleteDirectory(dir) {
```

Comment:
```
const fs = require('fs')

// directory path
const dir = 'temp'

// delete directory recursively
fs.rmdir(dir, { recursive: true }, err => {
  if (err) {
    throw err
  }

  console.log(`${dir} is deleted!`)
})
```

Reply:
Done with recursive call for removing the directory